src/store/bitpay-id/bitpay-id.effects.ts

```diff
@@ -119,10 +119,11 @@ export const startBitPayIdAnalyticsInit =
           agreedToMarketingCommunications,
         ),
       );
-      
       const systemEnabled = await checkNotificationsPermissions();
       if (systemEnabled && notificationsAccepted) {
-        dispatch(setNotifications(true));
+        Braze.setPushNotificationSubscriptionType(
```

Comment:
It's not necessary. `setNotifications(true)` already call to 
```
const value = accepted
      ? Braze.NotificationSubscriptionTypes.OPTED_IN
      : Braze.NotificationSubscriptionTypes.UNSUBSCRIBED;
```

Reply:
This is necessary because setNotifications was subscribing wallets to BWS each time the app was refreshed, which I didn't want.